src/pages/_app.tsx

```diff
@@ -20,7 +20,9 @@ export default function App({ Component, pageProps }: AppProps) {
   }, []);
 
   return (
-    <ChakraProvider theme={theme}>
+    // `colorModeManager` added to fix flashing issue
+    // See: https://chakra-ui.com/docs/styled-system/color-mode#add-colormodemanager-optional-for-ssr
+    <ChakraProvider theme={theme} colorModeManager={localStorageManager}>
```

Comment:
Ah, right! Thanks for fixing this, sorry I missed it during dark mode implementation.